webapp-django/crashstats/settings/test.py

```diff
@@ -23,10 +23,5 @@
     'django.contrib.auth.hashers.MD5PasswordHasher',
 )
 
-# Fix some implementations configuration
-SOCORRO_IMPLEMENTATIONS_CONFIG['resource']['boto'] = {  # noqa
-    'bucket_name': 'crashstats-test'
-}
```

Comment:
This was all kinds of wrong and caused the tests to hang because it uses the local dev environment values for creating a boto ConnectionContext and that causes moto to be sad and fail.